config/tips.txt

```diff
@@ -91,27 +101,31 @@ As the Chef, you can load your food into snack vending machines.
+As a Shaft Miner, the west side of the asteroid has a lot more minerals than on the east, but also contains monsters.
+As a Shaft Miner, every monster on the west side of the asteroid has a pattern you can exploit to minimize damage from the encounters.
+As a Shaft Miner, you can harvest goliath plates from goliaths and upgrade your rigsuit with them, greatly reducing incoming melee damage.
+As a Shaft Miner, you should always turn your jumpsuit sensors to maximum, so a fellow miner or cyborg can come to save you if you die.
 As a Traitor, the cryptographic sequencer (emag) can not only open doors, but also lockers, crates, APCs and more. It can hack cyborgs, and even cause bots to go berserk. Experiment!
 As a Traitor, subverting the AI to serve you can make it an extremely powerful ally. However, be careful of the wording in the laws you give it, as it may use your poorly written laws against you!
 As a Traitor, the Captain and the Head of Security are two of the most difficult to kill targets on the station. If either one is your target, plan carefully.
 As a Traitor, you can manufacture and recycle revolver bullets at a hacked autolathe, making the revolver an extremely powerful tool.
 As a Traitor, you may sometimes be assigned to hunt other traitors, and in turn be hunted by others.
+As a Traitor, the syndicate encryption key is very useful for coordinating plans with your fellow traitors -- or, of course, betraying them.
 As a Nuclear Operative, communication is key! Use :t or :h to speak to your fellow operatives and coordinate an attack plan.
-As a Nuclear Operative, you should look into purchasing a syndicate cyborg, as they can provide heavy fire support, full access, and are immune to conventional stuns.
-As a Nuclear Operative, stick together! While your equipment may save you from a lucky disarm or skillful taser bolt, your fellow operative are much better at saving your life: they can drag you away from danger while stunned and provide cover fire.
+As a Nuclear Operative, you should look into purchasing a syndicate cyborg, as they can provide heavy fire support, full access, are immune to conventional stuns, and can easily take down the AI.
+As a Nuclear Operative, stick together! While your equipment is robust, your fellow operatives are much better at saving your life: they can drag you away from danger while stunned and provide cover fire.
 As a Nuclear Operative, remember that capturing the disk and arming the Nuke is your task, don't just wander off and kill random people!
 As a Nuclear Operative, you might end up in a situation where the AI has bolted you into a room. Having some spare C4 in your pocket can save your life.
 As a Monkey, you can crawl through air or scrubber vents by alt+left clicking them. You must drop everything you are wearing and holding to do this, however.
 As a Monkey, you can still wear a few human items, like backpacks and gas masks, and still have two free hands. Remember that you cannot use guns!
```

Comment:
because tips are not for you, they are for newer players mostly (this goes for 75% of comments here)